WireUI/Sources/WireIndividualToTeamMigrationUI/Views/Previews/_FeaturePreview.swift

```diff
@@ -0,0 +1,74 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import WireDomainAPI
+
+struct MockUseCase: IndividualToTeamMigrationUseCase {
+    func invoke(teamName: String) async throws -> IndividualToTeamMigrationResult {
+        try await Task.sleep(nanoseconds: NSEC_PER_SEC * 5)
+        return IndividualToTeamMigrationResult(teamID: UUID(), teamName: teamName)
+    }
+}
+
+private struct FeaturePreviewContainer: UIViewControllerRepresentable {
+
+    typealias UIViewControllerType = IndividualToTeamMigrationViewController
+    let features: [TeamPlanFeature]
+
+    func makeUIViewController(context: Context) -> IndividualToTeamMigrationViewController {
+
+        IndividualToTeamMigrationViewController(
+            features: features,
+            useCase: MockUseCase(),
+            actionCallback: { _ in }
+        )
+    }
+
+    func updateUIViewController(_ uiViewController: IndividualToTeamMigrationViewController, context: Context) {
+        // Updates the state of the specified view controller with new information from SwiftUI.
+    }
+}
+
+#Preview {
+    FeaturePreviewContainer(features: [
```

Comment:
I briefly tried to set the font sizes larger and noticed that the view is not usable anymore. Also the content is clipped at the top and the bottom.
Does this issue only exist in previews?
![Screenshot 2024-12-04 at 13 10 51](https://github.com/user-attachments/assets/764b8186-608f-4f9b-8fff-18c211bcb1c2)

Reply:
We could wrap it all in a scroll view.

Reply:
done: https://github.com/wireapp/wire-ios/pull/2135/commits/1dea503e8a8ea8b9475d88e3e2afa358e815d3ff